Packages/com.createneptune.sdk/Runtime/CNWebUtils.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace CreateNeptune
+{
+    public static class CNWebUtils
+    {
+        private static readonly HashSet<string> IgnoredErrors = new HashSet<string>
+        {
+            "Host could not be resolved",
```

Comment:
The strings to check for here are:
* "Cannot resolve destination host"
* "SSL CA certificate error"
* "Unable to complete SSL connection"
* "Unknown error"
* "Cannot connect to destination host"
* "Request timeout"